src/material-experimental/mdc-snack-bar/snack-bar-container.scss

```diff
@@ -16,6 +16,11 @@
 }
 
 // These elements need to have full width using flex layout.
-.mat-mdc-snack-bar-handset, .mat-mdc-snack-bar-container, .mat-mdc-snack-bar-label {
-  flex: 1;
+.mat-mdc-snack-bar-handset,
+.mat-mdc-snack-bar-container,
+.mat-mdc-snack-bar-label {
+  // We use the individual properties, instead of the
```

Comment:
I think we can still use `flex`, just make it `flex: 1 0` which sets the grow and shrink properties at the same time

Reply:
That didn't work, but setting all the values did (`flex: 1 0 auto`). Changed.